src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -182,35 +182,32 @@ function bindCopyButton() {
         return false;
     });
 }
+var prevRow;
 
 function bindCopyEvents() {
-
-    $('#copyTableModal >tbody>tr').on('click', function(e) {
+    $('#copyTableModal > tbody > tr').on('click', function(e){
         e.preventDefault();
-
+        
         var currentRow = $(this);
-        if (currentRow.hasClass('row-selected')) {
-            return;
+        
+        if (typeof prevRow != 'undefined') {
+            $(prevRow).removeClass('row-selected');
+            $($(prevRow).children('td:first')).html('<input type="radio">');
         }
-
+        prevRow = currentRow;
+        
         var cells = currentRow.children('td');
         var courseId = $(cells[1]).text().trim();
         var feedbackSessionName = $(cells[2]).text().trim();
+        
         $('#modalCourseId').val(courseId);
         $('#modalSessionName').val(feedbackSessionName);
-
-        var selectedRadio = currentRow.parent().find('input:checked');
-        var selectedRow = selectedRadio.parent().parent();
-
-        selectedRadio.prop('checked', false);
-        selectedRow.removeClass('row-selected');
-
-        selectedRadio = currentRow.children('td').children('input');
-        selectedRadio.prop('checked', true);
+                
         currentRow.addClass('row-selected');
+        $(currentRow.children('td:first')).html('<input type="radio" checked="checked">');
```

Comment:
the previous way of doing selectedRadio.prop('checked', false) is way faster than creating a DOM element.

![screen shot 2015-06-18 at 8 04 48 pm](https://cloud.githubusercontent.com/assets/6291947/8230706/4f33caac-15f5-11e5-9805-4420163a574d.png)


![screen shot 2015-06-18 at 8 04 40 pm](https://cloud.githubusercontent.com/assets/6291947/8230707/4f5b5338-15f5-11e5-91a6-1609c5fab525.png)

In general, when it is possible not to create DOM elements, try not to, creating DOM elements via JavaScript is always slower than changing attributes. And looking at the test runners above, it is at least 70 times slower, usually about ~100 times slower.